src/System.Globalization/tests/CultureInfo/CultureInfoClone.cs

```diff
@@ -25,5 +25,15 @@ public void Clone(CultureInfo culture)
             Assert.Equal(culture, clone);
             Assert.NotSame(clone, culture);
         }
+
+        [Fact]
+        public void TestCalendarAfterCloning()
+        {
+            CultureInfo ci = new CultureInfo("en-US");
+            Assert.True(ci.Calendar == ci.DateTimeFormat.Calendar);
+            CultureInfo ci1 = (CultureInfo) ci.Clone();
```

Comment:
can you also add a check for `ci.Clone().Calendar != ci.Clone().Calendar`

Reply:
Isn't it the last assert I have? 

Assert.True(ci.Calendar != ci1.Calendar);

Reply:
oh, sorry I misread your comment. I'll add this